packages/js/src/ws/socket.ts

```diff
@@ -87,13 +87,17 @@ export class Socket extends BaseModule {
 
   initialize(): void {
     if (this.#token) {
-      this.#initializeSocket().then((error) => console.error(error));
+      this.#initializeSocket().catch((error) => {
```

Comment:
noticed this small mistake that was causing the `undefined` error in the console